sdk/eventhub/event-hubs/samples/sendEvents.ts

```diff
@@ -42,18 +40,36 @@ async function main(): Promise<void> {
     //   { body: { "message": "Hello World 2" } },
     //   { body: { "message": "Hello World 3" } }
     // ];
+    console.log("Sending single event...");
+    const scientist = listOfScientists[0];
+    producer.send({ body: `${scientist.firstName} ${scientist.name}` });
+
+    console.log("Sending batch events...");
```

Comment:
We shouldn't use "batch" here, it will get confused with the other example of "batch"

```suggestion
    console.log("Sending multiple events...");
```